src/core/Form/Select/SingleSelect/SingleSelect.tsx

```diff
@@ -146,7 +146,7 @@ class BaseSingleSelect<T> extends Component<
       const resolvedSelectedItem =
         'selectedItem' in nextProps ? selectedItem : prevState.selectedItem;
       const resolvedInputValue = selectedItemChanged
-        ? resolvedSelectedItem?.labelText
+        ? resolvedSelectedItem?.labelText || ''
```

Comment:
Just a thought: should we add a test for this?

Reply:
Added!

Reply:
Added!